cli/runtime/config/cli_discovery_sources_test.go

```diff
@@ -339,9 +342,46 @@ func TestDeleteCLIDiscoverySource(t *testing.T) {
 					},
 				},
 			},
+			count:   1,
 			input:   "test",
 			deleted: true,
 		},
+		{
+			name: "should delete local source",
+			src: &configapi.ClientConfig{
+				ClientOptions: &configapi.ClientOptions{
+					CLI: &configapi.CLIOptions{
+						DiscoverySources: []configapi.PluginDiscovery{
+							{
+								GCP: &configapi.GCPDiscovery{
+									Name:         "test2",
+									Bucket:       "test-bucket2",
+									ManifestPath: "test-manifest-path2",
+								},
+								ContextType: configapi.CtxTypeTMC,
+							},
+							{
+								Local: &configapi.LocalDiscovery{
+									Name: "default",
+									Path: "standalone",
+								},
+								ContextType: configapi.CtxTypeK8s,
+							},
+							{
+								Local: &configapi.LocalDiscovery{
+									Name: "admin-local",
```

Comment:
Looks like we support sources of same name of different discovery type? If so, would be good to include a test to verify the expected behavior when delete is issued on the name.

Reply:
> Looks like we support sources of same name of different discovery type? 

I don't think we support this AFAIK.

Reply:
discovery source are unique by name itself; Cannot have same name for two different types. There should only be one entry matching with the name.